src/Our.Umbraco.InnerContent/Web/UI/App_Plugins/InnerContent/views/innercontent.doctypepicker.html

```diff
@@ -1,42 +1,38 @@
-                    </th>
-                    <th>
-                        Name Template
-                        <i class="icon icon-help-alt inner-content__help-icon"
-                           data-tooltip="Enter an angularjs expression to evaluate against each item for its name."
-                           ng-mouseover="vm.tooltipMouseOver($event)"
-                           ng-mouseleave="vm.tooltipMouseLeave()"></i>
-                    </th>
-                    <th class="td-min" ng-show="model.value.length > 1" />
-                </tr>
-            </thead>
-            <tbody ui-sortable="vm.sortableOptions" ng-model="model.value">
-                <tr ng-repeat="config in model.value">
-                    <td class="icon icon-navigation"></td>
-                    <td>
-                        <select ng-options="dt.guid as dt.name for dt in vm.docTypes"
-                                ng-model="config.icContentTypeGuid" required></select>
-                    </td>
-                    <td>
-                        <input type="text" ng-model="config.nameTemplate" />
-                    </td>
-                    <td ng-show="model.value.length > 1">
-                        <a class="btn btn-danger" ng-click="vm.remove($index)">Remove</a>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-        <div>
-            <a href="" class="btn" ng-click="vm.add()">Add</a>
+    <div class="inner-content__help-container">
+        <i class="icon icon-help-alt inner-content__help-icon"
+           data-tooltip="Name Template: Enter an angularjs expression to evaluate against each item for its name."
+           ng-mouseover="vm.tooltipMouseOver($event)"
+           ng-mouseleave="vm.tooltipMouseLeave()"></i>
+    </div>
+    <div class="list-view-layouts" ui-sortable="vm.sortableOptions" ng-model="model.value">
+        <div class="list-view-layout" ng-repeat="config in model.value">
+            <i class="icon-navigation list-view-layout__sort-handle"></i>
+            <div>
+                <a href="" ng-click="vm.openDocTypePicker(config)" class="list-view-layout__icon">
+                    <i class="{{ vm.getContentType(config.icContentTypeGuid).icon ||  'icon-stop' }}"></i>
+                </a>
+            </div>
+            <div class="list-view-layout__name">
+                <span class="list-view-layout__name-text">{{ vm.getContentType(config.icContentTypeGuid).name || 'Name' }}</span>
+                <span class="list-view-layout__system">({{ vm.getContentType(config.icContentTypeGuid).alias || 'alias' }})</span>
+            </div>
+            <div class="list-view-layout__path">
+                <input type="text" ng-model="config.nameTemplate" placeholder="name template..." class="-full-width-input" />
+            </div>
+            <div>
+                <div class="list-view-layout__remove">
+                    <i class="icon-trash" ng-click="vm.remove($index)"></i>
```

Comment:
I'd like to use the `<umb-confirm-action>` directive for removals.
Umbraco's docs: https://our.umbraco.com/apidocs/ui/#/api/umbraco.directives.directive:umbConfirmAction